mava/configs/env/lbf_scenario/Foraging-10x10-3p-3f.yaml

```diff
@@ -0,0 +1,9 @@
+# The config of the Foraging-10x10-3p-3f scenario
+
+task_config:
```

Comment:
A general comment on the config naming. I think it will be nicer to to remove `Foraging-` from all the LBF scenario names since all the scenario configs are already in the `lbf_scenario` folder.

Reply:
Sure I'll change them it makes the name longer and it's redundant!